manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -472,6 +472,10 @@ data:
       {{- if .Values.global.logAsJson }}
         - --log_as_json
       {{- end }}
+      {{ if gt .ProxyConfig.Concurrency 0 -}}
+        - --concurrency
+        - "{{ .ProxyConfig.Concurrency }}"
+      {{ end -}}
```

Comment:
gt to ge ?
Concurrenty == 0 means concurrency == 1 at this moment. 
Unless the `.ProxyConfig.Concurrency == 0` means `use all cores` at istio

Reply:
The PR seems fixed the issue. If possible, istio should reduce the gap between envoy when the config name looks the same

Reply:
This is actually a behavior change from 1.5 right? I think it is a good change but we should document that in release note. concurrency=0 use to mean num of cpu workers, not it will be bound to proxy cpu limit.

Reply:
This is copy pasted from 1.5, I don't think it changed behavior. I think that only applied to gateway?

Reply:
Ok yeah, confirmed it is the same behavior in 1.5. thanks!